Index: .vscode/launch.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n    // Use IntelliSense to learn about possible attributes.\r\n    // Hover to view descriptions of existing attributes.\r\n    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387\r\n    \"version\": \"0.2.0\",\r\n    \"configurations\": [\r\n        {\r\n            \"name\": \"Python: Current File\",\r\n            \"type\": \"python\",\r\n            \"request\": \"launch\",\r\n            \"program\": \"${file}\",\r\n            \"console\": \"integratedTerminal\",\r\n            \"justMyCode\": true,\r\n            \"args\" : [\"C:\\\\Users\\\\41764\\\\Documents\\\\GitHub\\\\project_pcl_mturra_mkessle\\\\Books\\\\DrJekyllAndMrHyde.txt\"]\r\n        }\r\n    ]\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.vscode/launch.json b/.vscode/launch.json
--- a/.vscode/launch.json	(revision dc9f6dd4e3d251ab3f94271d779f8278c2fe0104)
+++ b/.vscode/launch.json	(date 1702112489709)
@@ -11,7 +11,7 @@
             "program": "${file}",
             "console": "integratedTerminal",
             "justMyCode": true,
-            "args" : ["C:\\Users\\41764\\Documents\\GitHub\\project_pcl_mturra_mkessle\\Books\\DrJekyllAndMrHyde.txt"]
+            "args" : ["C:\\Users\\marie\\Documents\\HS2023\\PCL1\\project_pcl_mturra_marikes\\TheStrangeCase_Gutenberg.txt"]
         }
     ]
 }
\ No newline at end of file
